ui/src/router/components/form/components/ensembler_config/RouteSelectionPanel.js

```diff
@@ -1,48 +1,78 @@
 import React, { useEffect } from "react";
-import { EuiFlexItem, EuiForm, EuiFormRow } from "@elastic/eui";
+import { EuiFlexItem, EuiForm, EuiFormRow, EuiSuperSelect } from "@elastic/eui";
 
+import { RouteDropDownOption } from "../RouteDropDownOption";
 import { Panel } from "../Panel";
-import { EuiComboBoxSelect } from "../../../../../components/form/combo_box/EuiComboBoxSelect";
 
 const SelectRouteIdCombobox = ({ value, routeOptions, onChange, ...props }) => {
-  // Note: routeNames changes on every render, which we need to capture name changes to routes
-  const routeNames = routeOptions.map((e) => (!!e.value ? e.value : e.label));
+  const noneValue = "_none_";
```

Comment:
As captured in the inline code comments below, this special value (can be anything) is needed as a default, to select a placeholder-like option since the `EuiSuperSelect` doesn't have a placeholder property yet. The same mechanism is used in the traffic rules panel.